quantum/stm32/mcuconf.h

```diff
@@ -187,7 +187,8 @@
  * PWM driver system settings.
  */
 #define STM32_PWM_USE_ADVANCED FALSE
-#define STM32_PWM_USE_TIM1 FALSE
+/* PWMD1 used by quantum/audio/audio_arm_pwm.c */
```

Comment:
This comment will automatically be removed next time we upgrade ChibiOS, no point keeping it here.
```suggestion
```

Reply:
aw - ok, we'll drop that comment then